Web/starter_project_iconics/components/about/Card.tsx

```diff
@@ -0,0 +1,67 @@
+
+import Carditems from "./CardItems";
+
+
+function Cards() {
+    return (
+        <div className="cards">
+            <div className="cards__container">
+                <div className="cards__wrapper">
+                    <ul className="cards__items  grid grid-cols-3 grid-rows-2">
+                        <div className="card__item  rounded-3xl shadow-md m-2 p-4">
+                            <Carditems 
```

Comment:
You should store this in an array and loop over them. This code seems a bit redundant.